tests/test_html.py

```diff
@@ -175,6 +175,29 @@ def content_summary():
     )
 
 
+def test_text_to_html_takes_one_string_value_and_returns_markup():
+    assert text_to_html("") == Markup("")
+    assert text_to_html("Hello World") == Markup("Hello World")
+
```

Comment:
```suggestion
assert text_to_html("hello world") == Markup("hello world")
```

To check it doesn't capitalise first if no kwargs passed?

Reply:
Good idea